src/mappings/omim.sssom.tsv

```diff
@@ -7826,6 +7825,7 @@ MONDO:0015855		skos:exactMatch	omim.ps:113700	Breasts and/or nipples, aplasia or
 MONDO:0015912		skos:exactMatch	OMIM:155100	macrothrombocytopenia and granulocyte inclusions with or without nephritis or sensorineural hearing loss	semapv:UnspecifiedMatching
 MONDO:0015942		skos:exactMatch	omim.ps:305620	Frontometaphyseal dysplasia	semapv:UnspecifiedMatching
 MONDO:0015977		skos:exactMatch	omim.ps:601495	Agammaglobulinemia	semapv:UnspecifiedMatching
+MONDO:0015991		skos:exactMatch	omim.ps:215700	Citrullinemia	semapv:UnspecifiedMatching
```

Comment:
## Tangent: Why `omim.ps` CURIE?
@twhetzel I see that this is what you were talking about at our meeting earlier.

@matentzn This doesn't seem related to what Trish did. When I do a global search, this file is the only one in `mondo-ingest` with `omim.ps`. So I take it that this is coming from `mondo`? Should we create an issue to address this there?

- [ ] Consider making issue to address `omim.ps`